logwatch: add page

kalebo opened the pull request:
Logwatch is a useful command to summarize log files. It can be configured to use settings from it's config file and run by cron, but it can also be useful to run ad-hoc from the command line.

agnivade commented on pages/linux/logwatch.md:
```diff
@@ -0,0 +1,11 @@
+# logwatch
+
+> Analyzes and summarizes system logs.
```
What does "system" mean here ? syslog ? Or any application log ? If its the latter, where is the location specified ?

agnivade commented on pages/linux/logwatch.md:
```diff
@@ -0,0 +1,11 @@
+
+- Analyze yesterday's logs in medium detail:
+
+`logwatch --range yesterday --detail med`
```
yesterday and med to be in tokens.

agnivade commented on pages/linux/logwatch.md:
```diff
@@ -0,0 +1,11 @@
+
+- Get the summarized login information for the last 3 weeks:
+
+`logwatch --range 'since 3 weeks ago' --detail low --service pam_unix `
```
Please use tokens for the 3 options. And I think, instead of specifying med and low in separate examples, maybe you can mention 'in a given detail' in the description and write '--details {{low|medium|others}}' in the example. A shorter and concise way to express more.

waldyrious commented:
Reopening as the "previous prompt" wasn't provided by @probot-stale. Please see #1350 and #1360 for details.

kalebo commented:
@agnivade I addressed the issues you mentioned in the review; however, I'm not sure how to make the commit I just added to switch the issue status from "Changes requested" back to "Needing Review".

waldyrious commented:
@kalebo I don't think github does that, though indeed it should (the reviewing flow will most likely have these rough edges polished over time). In any case, @agnivade will probably take a look soon :)

agnivade commented on pages/linux/logwatch.md:
```diff
@@ -1,11 +1,7 @@
 
-`logwatch --range yesterday --detail med`
-
-- Get the summarized login information for the last 3 weeks:
```
I think the `--service` flag is a useful one. In case someone wants to get logs of a particular service. Lets keep that one.

agnivade commented on pages/linux/logwatch.md:
```diff
@@ -1,11 +1,7 @@
 # logwatch
 
-> Analyzes and summarizes system logs.
+> Summarizes many different logs for common services (e.g., apache, SSHD, etc.) in single report.
```
-> in _a_ single report.

waldyrious commented:
Ping @kalebo :) (let us know you feel we're being unreasonably picky, or have any other concerns -- we're still learning the kinks of properly maintaining such a highly collaborative project).

waldyrious commented:
Thanks for the contribution! 🎉
